src/lib/tasks/Login.ts

```diff
@@ -193,4 +223,33 @@ export class LoginTask extends Task<LoginTaskParams> {
     }
     log.debug('Login after redirections', { pageUrl: page.url(), chain });
   }
+
+  async #handleSpecForm(): Promise<void> {
```

Comment:
Spec? Doesn't ring a bell for me, if you can explicit the name

Reply:
"spec for special case" Do you have something better?
Maybe `handleFormSpecialCase` works?